src/content/docs/en/guides/rss.mdx

```diff
@@ -214,6 +214,30 @@ export function GET(context) {
 }
 ```
 
+## Removing trailing slashes
+
+RSS feed includes links with a trailing slash by default. You can change that behavior by setting the `trailingSlash` option when calling `rss`.
+
+If you have set `trailingSlash: "never"` on your `astro.config.mjs`, set `trailingSlash: false` on the call to `rss` so your feed matches your project configuration.
```

Comment:
```suggestion
Astro's RSS feed produces links with a trailing slash by default, no matter what value you have configured for  `trailingSlash`. This means that your RSS links may not match your post URLs exactly.
 
If you have set `trailingSlash: "never"` on your `astro.config.mjs`, set `trailingSlash: false` in the `rss()` helper so that your feed matches your project configuration.
```